wildfly/connectors/salesforce/feature-pack-salesforce/src/main/resources/modules/system/layers/dv/org/jboss/teiid/resource-adapter/salesforce/main/module.xml

```diff
@@ -18,7 +18,7 @@
         <module name="javax.jws.api"/>
         <module name="javax.wsdl4j.api"/>
         <module name="org.jboss.teiid.translator.salesforce"/>
-        <module name="org.apache.cxf.impl.frontend-jaxrs" services="export" export="true"/>
+<!--        <module name="org.apache.cxf.impl.frontend-jaxrs" services="export" export="true"/>-->
```

Comment:
I've noticed this commented out in a few `module.xml` files (at least this and the webservice one). Is this a temporary commenting is this dependency obsolete now?